src/components/MovieCard/MovieCard.scss

```diff
@@ -55,16 +57,14 @@
       border: 1px solid #979797;
       opacity: 0.5;
       border-radius: 4px;
+      @include font($weight: 500, $size: 14px, $height: 17px);
+      text-align: center;
+    }
 
-      p {
-        @include font($weight: 500, $size: 14px, $height: 17px);
-        text-align: center;
-      }
+    &__genres {
+      @include font($weight: 500, $size: 14px, $height: 17px);
+      margin-top: 8px;
+      opacity: 0.7;
```

Comment:
If you're striving to achieve some text color don't use opacity for that, use the color instead. I saw that in a lot of places. Please scan the app and replace them.